programs/protocol/src/data/carrier.rs

```diff
@@ -15,6 +15,10 @@ pub struct Availability {
 #[account(zero_copy)]
 #[derive(Debug, Default, PartialEq, BorshSerialize, BorshDeserialize)]
 pub struct Carrier {
+    pub creator: Pubkey,
     pub authority: Pubkey,
+    pub name: Name,
     pub availability: Availability,
+    pub offers: u32,
```

Comment:
count lost meaning i guess

Reply:
offers are offers to accept shipments 
count is a count of accepted shipments

Reply:
but isnt better to call it offers_count accepted_offers_count or smth

Reply:
count is meaningless

Reply:
right, will do